pypowerwall/__init__.py

```diff
@@ -179,6 +180,8 @@ def __init__(self, host="", password="", email="nobody@nowhere.com",
             self.cloudmode = True
         if self.cloudmode and not self.fleetapi:
             self.mode = "cloud"
+        elif self.cloudmode and self.fleetapi:
+            self.mode = "fleetapi"
```

Comment:
Without this, there was no way to force fleetapi mode except via auto_select. Now you can test fleetapi with:

```python
PW_HOST=""
PW_EMAIL="example@example.com"

import pypowerwall
pw = pypowerwall.Powerwall(host=PW_HOST, email=PW_EMAIL, fleetapi=True)
print(f"mode: {pw.mode}")
pw.power()
```